incubator/kubevirt/README.md

```diff
@@ -0,0 +1,45 @@
+# KubeVirt Helm Chart
+It will deploy [KubeVirt](https://github.com/kubevirt/kubevirt) resources to make possible to create VM's as part of k8s environment.
+You can find more information about VM yaml specification under [KubeVirt User Guide](https://kubevirt.gitbooks.io/user-guide/)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+```
+
+You will need to give additional permissions to the user `system:serviceaccount:kube-system:default`, it will be valid only for cases when you initiated `helm` under `kube-system` namespace.
+
+```
+$ kubectl create clusterrolebinding --user system:serviceaccount:kube-system:default kube-system-cluster-admin --clusterrole cluster-admin
+```
+
+Deploy KubeVirt chart.
+```
+$ helm install --name my-release incubator/kubevirt
+```
+
+After installation finish, it better to remove extra permissions from the system.
+
+```
+$ kubectl delete clusterrolebinding kube-system-cluster-admin
+```
+
+## Deleting the Charts
+
+Delete the Helm deployment as normal
+
+```
+$ helm delete my-release
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the kubevirt chart and their default values.
+
+|                Parameter                  |            Description              |          Default           |
+| ----------------------------------------- | ------------------------------------| -------------------------- |
+| `kubevirt.repository`                     |haproxy container name               | `kubevirt`                 |
+| `kubevirt.tag`                            |haproxy container tag                | `latest`                   |
```

Comment:
The two parameteres are not specific to haproxy, but used for all images

as the chart gives version 0.1.0, the tag should also default to this version.

Reply:
Missed this one